2-mandatory/4-restaurant.js

```diff
@@ -21,7 +21,21 @@ const MENU = {
 
 let cashRegister = {
   // write code here
-}
+  orderBurger(balance) {
```

Comment:
This is great :) Can you think of a way to avoid duplicating the price of the burger and falafel? Can you think of a reason we'd want to avoid duplicating the price all over the code?